modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase

```diff
@@ -529,17 +529,27 @@ definition {
 	@description = "LPS-142175 - Verify it's possible to import object with Actions"
 	@priority = 4
 	test CanImportObjectWithAction {
-		task ("When import object with action") {
+		task ("Given a imported object with action") {
 			ObjectExportImport.importObject(
 				objectFileName = "ImportedObjectWithAction",
 				objectName = "ImportedObjectWithAction160652");
 		}
 
-		task ("Then assert the imported file is present") {
+		task ("When assert the imported file is present") {
 			AssertElementPresent(
 				key_labelName = "Imported Object With Action",
 				locator1 = "CreateObject#OBJECT_NAME");
 		}
+
+		task ("Then the action is present on the object") {
+			ObjectPortlet.selectCustomObject(label = "Imported Object With Action");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.assertObjectActions(
+				actionActive = "Yes",
+				actionLabel = "CanImportObjectWithAction");
+		}
```

Comment:
Hey @JhossephG, looks good! Just a couple of feedback:

Your action here is importing the object, so any step that does that will be under `When`. All assertions after the action will be under `Then`. If there are no requirements to import an object, then I think it's ok to skip the `Given`. Let me know what you think! Thank you!

Reply:
Make sense @patriciajeanperez, I'll update the tasks!